### ISSUE TITLE
Side effect of #6526

### ISSUE THREAD

**Issue by ellisonbg** [jupyterlab/jupyterlab#6532]
**Describe the bug**

#6526 introduced 2 new problems:

* Border at top of panel in main work area went away altogether.
* Close icon in tab of main work area is too high.

**Screenshots**

![Screen Shot 2019-06-10 at 6 21 27 PM](https://user-images.githubusercontent.com/27600/59236895-ebe80d00-8bac-11e9-96ae-4522112b0762.png)

**Comment by ian-r-rose** [comment]
I don't think #6526 is the culprit of the close icon issue. It doesn't appear in my testing, nor in the screenshot on that PR.

**Comment by ellisonbg** [comment]
@blink1073 this is probably related to the big CSS refactor then...

**Comment by ian-r-rose** [comment]
The border seems to be due to #6526, however. Perhaps we restore the tabBar border, then cover it up with an individual tab border on the active tab?

**Comment by jasongrout** [comment]
I tried doing:

```diff
diff --git a/packages/application/style/tabs.css b/packages/application/style/tabs.css
index ab3b9f0c5..14569d800 100644
--- a/packages/application/style/tabs.css
+++ b/packages/application/style/tabs.css
@@ -19,6 +19,8 @@
 |----------------------------------------------------------------------------*/
 
 .p-DockPanel-tabBar {
+  border-bottom: var(--jp-border-width) solid var(--jp-border-color1);
+  overflow-y: visible;
   color: var(--jp-ui-font-color1);
   font-size: var(--jp-ui-font-size1);
 }
```

but according to https://developer.mozilla.org/en-US/docs/Web/CSS/overflow:

> Setting one axis to visible (the default) while setting the other to a different value results in visible behaving as auto.

That means that a scroll bar appears in the right side. I think this has to do with anything other than overflow visible creates a new graphics context.

**Comment by jasongrout** [comment]
Fixed in #6548.
